iceoryx_posh/include/iceoryx_posh/popo/request.hpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+// Copyright (c) 2021 by AVIN Systems Private Limited All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef IOX_POSH_POPO_REQUEST_HPP
+#define IOX_POSH_POPO_REQUEST_HPP
+
+#include "iceoryx_posh/popo/smart_chunk.hpp"
+
+namespace iox
+{
+namespace popo
+{
+template <typename T, typename H>
+class RpcInterface;
+
+///
+/// @brief The Request class is a mutable abstraction over types which are written to loaned shared memory.
+/// These requests are publishable to the iceoryx system.
+///
+template <typename T, typename H = cxx::add_const_conditionally_t<mepoo::NoUserHeader, T>>
```

Comment:
please remove the second template parameter. The `Request` always has a `RequestHeader` and should not be changed by the user.